0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -490,7 +494,7 @@
 
 # Create list of questionable channel correlations (alignments)
 correlation_col_prefix = "Correlation_Correlation_"
-corr_base_cols = image_meta_col_list.copy() + ["site_location"]
+corr_base_cols = image_meta_col_list.copy() + ["site"]
```

Comment:
I don't think we want this change right? Our plots should include the number location, not the full string, right?  discussed https://github.com/broadinstitute/pooled-cell-painting-profiling-recipe/issues/45#issuecomment-670206381

Reply:
Drats. Sorry about that. I had switched to an older version so I could quickly generate qc metrics and then forgot to switch back. You are correct.